src/deployment/set_admins.py

```diff
@@ -5,12 +5,20 @@
 
 import argparse
 import json
+from time import sleep
 from typing import List, Optional
 from uuid import UUID
 
 from azure.common.client_factory import get_client_from_cli_profile
 from azure.cosmosdb.table.tableservice import TableService
 from azure.mgmt.storage import StorageManagementClient
+import logging
+
+## Disable logging from storageclient. This modules display and error message
+## when a resource is not found even if the exception is raised and handled internally.
```

Comment:
Should we open an issue in the upstream library?

Reply:
it is already being discussed
https://github.com/Azure/azure-storage-python/issues/516